lms/static/sass/multicourse/_dashboard.scss

```diff
@@ -801,13 +806,60 @@
       }
 
       .wrapper-extended {
-        padding: ($baseline/4) 0;
+        display: flex;
+        flex-flow: column wrap;
+
+        .wrapper-icon-message {
+          width: 100%;
+          display: inline-flex;
+          flex-wrap: wrap;
+
+          svg {
+            flex-shrink: 0;
+            width: 43px;
```

Comment:
Could be in `rem`? Might be an awkward decimal, in that case would leave as `px`

Reply:
Eugene wanted the icon to be as close to 32px x 32px as possible. I thought the idea was to keep the icon size constant and not scale.